content/en/docs/setup/production-environment/windows/upgrading-kubeadm-nodes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Upgrading Windows Nodes in Kubernetes
+min-kubernetes-server-version: v1.17
```

Comment:
Nice!
This works best if you also put a `{{< version-check >}}` shortcode in the prerequisites section.